grype/db/v5/store/store.go

```diff
@@ -4,14 +4,14 @@ import (
 	"fmt"
 	"sort"
 
+	_ "github.com/glebarez/sqlite" // provide the sqlite dialect to gorm via import
```

Comment:
not a comment for this PR: we should remove these statements and push these side-effect imports to the `main` package exclusively (as not to pollute the sql driver registration space for lib consumers).